### ISSUE TITLE
Validation loss feeds network twice

### ISSUE THREAD

**Issue by lgvaz** [airctic/icevision#18]
The modification previously made on `layers.ipynb` were affecting the model performance on evaluation. It's needed to better understand what is happening in `roi_heads.forward` before modifying the method.

For now it's okay to feed the model twice for getting the loss and then the predictions.

Getting the validation loss by using `model.train` also disconsiders other important effects like `Dropout` and `BatchNorm`
